docs/configuring-playbook-synapse.md

```diff
@@ -17,6 +17,51 @@ Alternatively, **if there is no pre-defined variable** for a Synapse setting you
 
 - or, if extending the configuration is still not powerful enough for your needs, you can **override the configuration completely** using `matrix_synapse_configuration` (or `matrix_synapse_configuration_yaml`). You can find information about this in [`roles/matrix-synapse/defaults/main.yml`](../roles/matrix-synapse/defaults/main.yml).
 
+## Synapse OpenID Connect
+
+If you plan to rely only on SSO and don't want your users to ever login with password credentials disable this:
```

Comment:
This part can be confusing, I think.

We say "disable this", but.. users don't have it explicitly enabled in their `vars.yml` file. Maybe they don't even know what it is.
It's not ready for copy-pasting, it needs to be adapted and you need to be careful to read the text above the code block.

Perhaps we should just put it somewhere else (down there with the other settings, below the bare-minimum config you've given). It could default to `true`, as it is here, and there could be a comment saying "you may wish to disable this".

Alternatively (even better, I think), it can be like this:

```yaml
# Some other settings here

# Authenticating against Synapse's localdb or other password providers is also possible by default.
# The former is controlled by `matrix_synapse_password_config_localdb_enabled`.
# Uncomment the line below to disable all authentication methods other than SSO.
# matrix_synapse_password_config_enabled: false
```